src/aws-code-deploy/orb.yml

```diff
@@ -212,7 +212,7 @@ commands:
               sleep 5
             done
 
-            if [[ $STATUS == "Ready" ]]; then
+            if [[ $STATUS == "Ready" || $STATUS == "Succeeded" ]]; then
```

Comment:
I'm not really sure if "Ready" means "deployments finished" or that loop above should retry until it will be "Succeeded"

Reply:
`Ready` is valid in blue/green deployments, when the new resources (_blue_) are ready to receive traffic but it didn't remove the old resources (_green_).
If you want to be on the safe side, you can wait for `Succeeded` every time.

Reply:
@kobim so in your opinion "Ready" should be moved to while loop above and this "if" should check only "Succeeded" status?

Reply:
I'm not using blue/green deployments that's why I'm not getting this "Ready" status.
Probably you are right that after "Ready" this deployment will change status to "Succeeded".
I found other sample that also checks only "Succeeded" status:
https://github.com/awslabs/aws-codedeploy-plugin/blob/master/src/main/java/com/amazonaws/codedeploy/AWSCodeDeployPublisher.java#L500

Reply:
@eddiewebb @felicianotech @iynere - Can we get some eyes on this PR?  I have hit on this issue as well, and can't really move forward with my CircleCI proof-of-concept for my team.